### ISSUE TITLE
uptimerobot influx creds trying to use configmap instead of secret

### ISSUE THREAD

**Issue by mattmattox** [k8s-at-home/charts#640]
# Details

**Helm chart name and version:**

k8s-at-home/uptimerobot:3.0.0

**Container name and tag:**

billimek/node-influx-uptimerobot:latest

**What steps did you take and what happened:**

```
 helm upgrade --install uptimerobot k8s-at-home/uptimerobot \
--set config.influxdb.host=1.2.3.4 \
--set config.influxdb.username=admin \
--set config.influxdb.password=password\
--set config.uptimerobot.apikey=123456789
```

**Relevant Helm values:**
```
            - name: INFLUX_USERNAME
              valueFrom:
                configMapKeyRef:
                  name: {{ template "uptimerobot.fullname" . }}
                  key: influxdb-username
            {{- end }}
            {{- if .Values.config.influxdb.password }}
            - name: INFLUX_PASSWORD
              valueFrom:
                configMapKeyRef:
                  name: {{ template "uptimerobot.fullname" . }}
                  key: influxdb-password
            {{- end }}
```
**What did you expect to happen:**

The values `INFLUX_USERNAME` and `INFLUX_PASSWORD` are stored in the secret uptimerobot and not the configmap

**Anything else you would like to add:**

Editing the deployment from `configMapKeyRef` to `secretKeyRef` resolves the issue.

**Additional Information:**

**Comment by mattmattox** [comment]
I submitted a PR to resolve this issue https://github.com/k8s-at-home/charts/pull/641

**Comment by bjw-s** [comment]
Closed with #641
